package.json

```diff
@@ -46,7 +46,7 @@
     "@types/boom": "^7.2.1",
     "@types/console-log-level": "^1.4.0",
     "@types/express": "^4.17.0",
-    "@types/hapi": "^18.0.2",
+    "@types/hapi__hapi": "^20.0.5",
```

Comment:
Why is this change happening?  The `hapi__hapi` thing is weird.  Was this due to the original package being renamed I guess?

Reply:
Yes, this is the official type import for @hapi/hapi now: 
`This package contains type definitions for @hapi/hapi (https://github.com/hapijs/hapi).`
from https://www.npmjs.com/package/@types/hapi__hapi